reference-implementation/web-platform-tests/readable-streams/general.js

```diff
@@ -85,7 +85,7 @@ test(() => {
     start(controller) {
       assert_equals(this, source, 'source is this during start');
 
-      const methods = ['close', 'enqueue', 'error', 'constructor'];
+      const methods = ['_cancel', '_pull', 'close', 'enqueue', 'error', 'constructor'];
```

Comment:
These shouldn't be here. I am not sure the best way to test them without breaking our reference implementation :(. Maybe use a `Symbol` for them in the implementation? That should work.